Documentation/project-docs/glossary.md

```diff
@@ -20,6 +20,7 @@ As much as possible, we should link to the most authoritative and recent source
 | JIT | [Just-in-Time](https://github.com/dotnet/coreclr/blob/master/Documentation/botr/ryujit-overview.md) compiler. RyuJIT is the code name for the next generation Just-in-Time(aka "JIT") for the .NET runtime. |
 | LCG | Lightweight Code Generation. An early name for [dynamic methods](https://github.com/dotnet/coreclr/blob/master/src/mscorlib/src/System/Reflection/Emit/DynamicMethod.cs). |
 | MD | MetaData. |
+| MDA | Managed Debugging Assistant - see [details](https://docs.microsoft.com/en-us/dotnet/framework/debug-trace-profile/diagnosing-errors-with-managed-debugging-assistants) |
```

Comment:
It maybe worth noting that MDAs are not in .NET Core. We are making the equivalent diagnostic functionality is available by other means where applicable.

Reply:
E.g. https://github.com/dotnet/coreclr/issues/15465

Reply:
Done